src/main/java/org/uwdigi/rag/config/ModelConfig.java

```diff
@@ -10,6 +11,7 @@
  * API keys/URLs.
  */
 @Configuration
+@Getter
```

Comment:
@gitcliff what is the getter for here?

Reply:
@reagan-meant That implicitly generates getters for all of the class properties, so that we're still able to call, e.g., `getGeminiApiKey()`, etc.

Reply:
Oh yeah, Sorry I thought we are not using it. I confused this bean with the AppConfig one.